src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main_header">
+      <a
+        href="/"
+        class="logo_img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav>
+        <ul class="device_nav nav__list">
```

Comment:
The class name 'device_nav' seems to describe the style rather than the meaning of the content. It's better to use class names that reflect the content's purpose or structure, like 'nav__list'.